### PR TITLE
HTML/CSS & JavaScript capstone project - Conference page

### PR DESCRIPTION
**Pull request by Dorcas126**
This is my first capstone project at Microverse which I implemented a conference page by using the following technologies:
 - HTML
 - CSS
 - JavaScript

## Learning objectives
- Understand how to use medium-fidelity wireframes to create a UI.
- Follow written requirements (e.g. user stories).
- Use semantic HTML tags.
- Apply best practices in HTML code.
- Use CSS selectors correctly.
- Use CSS box model.
- Use Flexbox to place elements in the page.
- Use images and backgrounds to enhance the look of a website.
- Demonstrate ability to create UIs adaptable to different screen sizes using media queries.
- Use GitHub Pages to deploy web pages.
- Apply JavaScript best practices and language style guides in code.
- Use JavaScript to manipulate DOM elements.
- Use JavaScript events.
- Use objects to store and access data.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+                            <div class="logo">
+                                <a href="index.html"><img class="my-logo" src="img/logo.png" alt=""></a>
+                            </div>
+                            <div class="hamburger">
+                                <div class="bar"></div>
+                                <div class="bar"></div>
+                                <div class="bar"></div>
+                            </div>
+                            <ul class="nav-list">
+                                <li class="nav-item about"><a href="about.html">About</a></li>
```
Comment (joel-kalema) [inline about.html]: - [ ]  You did great work @Dorcas126  😃, however, in this `about.html` page we are not supposed to have again the `about` button in your navbar. What do you think if we change it to this `<li class="nav-item about"><a href="index.html"> Home </a></li>` to facilitate the user to go back to the `home` page 👍. Kindly fix it and make sure it is the same for the `mobile menu` 💪.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                    </div>
+                </div>
+            </section>
+    
+            <section class="footer footer-about" id="footer">
+                <div class="main-container">
+                    <div id="footer-con">
+                        <div class="logo">
+                            <a href="index.html"><img class="my-logo4"src="img/creative-logo.png" alt=""></a>
+                            <img  class="md-footer-img" src="img/creative-logo2.png" alt="">
```
Comment (joel-kalema) [inline about.html]: - [ ] For the page's accessibility, Kindly make sure all images contain `alt` attributes 👍

File: index.html [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+                <span class="Program-span">Main Program</span>
+
+                <hr class="line">
+
+                <div class="Program-img">
+                    <div class="lecture">
+                        <img src="img/program_icon_01.png" alt="">
+                        <h3 class="text">Lecture</h3>
+                        <p class="p-tag">Listen to the speakers from various countries about the messages </p>
+                    </div>
```
Comment (joel-kalema) [inline index.html]: - [ ] Good job working on the activity section 👍, however, I'll suggest you display the `lecture` bloc as `display raw` to stick the design as much as possible 😃. 
- [ ] Also, what do you think if we make our `background` look like what we have on the `template`? You don't need to have the same image but let's get something that can have the color 👍 

> | | template | | Your project | |
> |-|---------|-|--------|-|
> | |Screenshot 2023-05-30 132715| |Screenshot 2023-05-30 132805| |

- [ ] Your section is good ❤️, however, let's add some `hover` to stick the design 💪.


Screenshot 2023-05-30 133904

File: index.html [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+        <section id="section2">
+            <div class="main-container">
+                <div class="presenters">
+                    <h2 class="main-program2">Featured Speakers</h2>
+                    <hr class="line">
+                    <div id="speakers-grid">
+                    </div>
+                    <div class="view-btn"><button id="viewmore">see more speakers</button></div>
+                </div>
+            </div>
```
Comment (joel-kalema) [inline index.html:139]: - [ ] Now let's talk about Speakers 😃. I like the way you have done using `JavaScript`,  However, currently, when we are clicking on the `see more speakers` button, we are just changing the value of the button, but we suppose to display all speakers then when we click again on `see less` we are supposed to display only two speakers. Kindly fix it 💪

File: style.css [hunk 5]
```diff
@@ -0,0 +1,584 @@
...
+.greeting {
+  color: rgb(221, 56, 56);
+  font-size: 20px;
+  margin-bottom: 10px;
+}
+
+.intro {
+  color: rgb(184, 3, 3);
+  font-size: 20px;
+}
```
Comment (joel-kalema) [inline style.css]: - [ ] Kindly make sure you stuck the design using a `background-image`, `font-size`, `font-weight` on your text, let's take an example from this.

```` buch
  color: transparent;
  font-size: 2.05rem;
  font-weight: 1000;
  text-transform: uppercase;
  background-image: url(....image url....);
  background-position: -700px;
  -webkit-background-clip: text;

````
#### Templete
Screenshot 2023-05-30 135309

#### Your design

Screenshot 2023-05-30 142349

File: README.md [hunk 6]
```diff
@@ -1 +1,179 @@
...
+Thank you.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments 
+
+I would like to thank each and everyone for your support and interest on my project.
+Thank you
```
Comment (joel-kalema) [inline README.md]: - [ ] Kindly make sure you gave your Acknowledgments to the Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07). as it is [required](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md#:~:text=Original%20design%20idea%20by%20Cindy%20Shin%20in%20Behance.) 😃

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @Dorcas126 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (Dorcas126) [reply index.html:139]: Thank you for your feedback, I appreciate it, but the `more` button functionality is not part of the project requirements. Thank you.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,240 @@
...
+                        </div>
+                    </div>
+
+                    <div class="lecture">
+                        <div class="lecture_icon">
+                          <img src="img/program_icon_05.png" alt="program picture">
+                          <h3 class="text">CC ignite</h3>
+                        </div>
+                        <div class="lecture_text">
+                          <p class="p-tag">Get opportnunities to network with CC affiliates around the world</p>
```
Comment (usorfaitheloho) [inline index.html:119]: As suggested by the previous reviewer, kindly use the background color that was used in the design given as we're always advised to make our design as really close as possible to any design given. For your design, you used a really dark colour while the design given has something similar to `#262932` for the background, kindly implement this change 😄

Design given : 
image

Your design :
image

File: index.html (hunk 4 above)
Comment (usorfaitheloho) [reply index.html:139]: Hi @Dorcas126 , the mobile view should also contain the same number of speakers as the desktop view, although the `more` button was not stated in the `requirements` it is however visible in the design and has a functionality, please kindly make the required changes for the `speaker` section and resubmit for a review 😄

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @Dorcas126  ,
> 
> Great job for implemented the some of the Project Requirement , but there are some changes that need to be implemented in order to proceed to the next project 👍
> 
> ## Highlights 👏🏻
> ✔️ Implemented review coments ✔️ You used correct Git Hub-Flow. ✔️ Good README ✔️ Good Commit Messages
> 
> almost 

> ## Required Changes ♻️
> _Check the comments below the review._
> 
> * [ ]  Kindly I recommend you to update your READEME file to include the **"Feature section "**  including 1-3 potential features to be added and please before push the READEME file to main branch kindly make sure that you update in your feature branch(here).
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+                </div>
+            </section>
+    
+            <section class="footer footer-about" id="footer">
+                <div class="main-container">
+                    <div id="footer-con">
+                        <div class="logo">
+                            <a href="index.html"><img class="my-logo4"src="img/creative-logo.png" alt="creative logo image"></a>
+                            <img  class="md-footer-img" src="img/creative-logo2.png" alt="creative logo image">
+                        </div>
```
Comment (mckent05) [inline about.html:110]: **OPTIONAL** - The sponsor section is missing in the mobile version of this page, though you have done well so far 👏, Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
awrhwrh
Design template:
awrgawrg

File: about.html [hunk 9]
```diff
@@ -0,0 +1,121 @@
...
+            </header>
+        </section>
+    
+        <main>
+            <section class="title-section">
+                <div class="main-container">
+                    <div class="page-title2">
+                        <article>
+                            <h3 class="greeting">"Hello! Learn about Photography"</h3>
+                            <h1 class="intro">CREATIVE ANNUAL GLOBAL LEARNING SUMMIT</h1>
```
Comment (mckent05) [inline about.html:52]: **OPTIONAL** - Please note that in order to improve the semantics of your page, I advise that your `hx` tag follow an ascending hierarchy i.e an `h2` tag should only be used after an `h1` tag. I suggest using a `p` tag in place of the `h3` tag. Please fix this 😄

**Review by mckent05 — APPROVED** [review]
Hi @Dorcas126, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- Good job creating your first capstone project, keep going 🚀 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**